e2e/autoscript-apps/src/authn-oauth/autoscript.ts

```diff
@@ -100,6 +103,8 @@ function autoscript() {
       mergeMap(
         (tokens) => {
           console.log('Force renew OAuth tokens');
+          // Pause for 2 seconds to allow expired tokens to expire
+          rxDelay(tokenExpiredDelay);
```

Comment:
same as below I think? @cerebrl maybe you can correct me here

Reply:
This delay was added by mistake, and will be removed